module4/intermission/index.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: page
+title: Intermission Work
+---
+
+Before starting Module 4, spend a minimum of 2 hours working through the following activities.
+
+We're going to start Module 4 by taking couple of days to adventure in Javascript world. The main focus of this course is backend development using C#, .NET, and SQL, but Javascript is such a key part of web development that's important for you to be familiar with the topic.
+
+### Part 1 (Required): Modifying Webpages with Javascript
+
+Start by reading the section [A High Level Definition](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/What_is_JavaScript#a_high-level_definition) in these MDN docs to get a sense of how Javascript fits in with HTML and CSS when building websites.
+
+Then move on to reading [What is the DOM](https://www.digitalocean.com/community/tutorials/introduction-to-the-dom#what-is-the-dom).
+
+Your goal is to learn how to change the background color of a favorite webpage to a new color using only Javascript written in the developer console.
+
+✅**Deliverable**✅ Take a screenshot of your result and send it to your instructors, it should look something like this:

```

Comment:
Maybe we should be more prescriptive of what webpage to change. We can limit to a webpage on `turing.edu` (there are a lot to choose from).

Reply:
I actually feel like it will be more impactful if they have the opportunity to change something on a random webpage that they visit in real life. Lots of options, but I think really any of the options should work.  But I also don't see a problem with them also changing the Turing background if they prefer.